### PR TITLE
Fixed aiming in house && bullets being destroyed in the house

### LINKED ISSUE #38: House collider destroys bullets in house

### PR DESCRIPTION
**Pull request by Twannes-Claes**

### CODE REVIEW

File: Kenneyjam23/Assets/Scripts/Player/Bullet.cs [hunk 1]
```diff
@@ -29,6 +29,9 @@ private void OnTriggerEnter(Collider other)
         {
             otherHealth.Damage(_damage);
         }
+
+        Debug.Log(other.gameObject.name);
```
Comment (Wardergrip) [inline Kenneyjam23/Assets/Scripts/Player/Bullet.cs]:
```suggestion
```

File: Kenneyjam23/Assets/Scripts/Player/PlayerController.cs [hunk 2]
```diff
@@ -104,6 +106,8 @@ private void RotatePlayer()
 
         Vector3 worldMousePos = hit.point;
 
+        //Debug.Log(hit.transform.name);
```
Comment (Wardergrip) [inline Kenneyjam23/Assets/Scripts/Player/PlayerController.cs]:
```suggestion  
```

**Review by Wardergrip — CHANGES_REQUESTED** [review]
Make sure debug.logs are gone before they get into main to prevent the console being clogged with unimportant log messages.

Will test functionality soon

**Comment by Wardergrip** [comment]
Aiming in house seems fine and bullets seem fine in house. So functionality looks fine to me.

**Review by Twannes-Claes — COMMENTED** [review]
Very good

**Pull request merged**